Add ISBN to book

```diff
@@ -0,0 +1,132 @@
+import java.util.Scanner;
+
+public class Test1 {
+
+   // method to check number is ISBN 
+   public static boolean isISBN(String number) {
+
+      // declare variable
+      int length = 0;
+
+      // remove all hyphens
+      number = number.replace("-", "");
+      // remove all spaces
+      number = number.replace(" ", "");
+
+      // check result string is a number or not
+      try {
```

Comment:
I would move the code that checks for a number in a different method so it can be tested separately.